internal/cmd/completion.go

```diff
@@ -0,0 +1,77 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+)
+
+func newCompletionsCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "completion",
+		Short: "Generate completions",
```

Comment:
```suggestion
		Short: "Generate shell auto-completion for the CLI",
```

I had a look at the help text on this branch, and in the list of commands it might not be too obvious what completions means. A few extra words might help put it in context.